adapters/kinetiq.ts

```diff
@@ -30,4 +30,7 @@ export default {
   }),
   total: (data: { points: number }) => ({ kPoints: data.points }),
   rank: (data: { rank?: number }) => data.rank || 0,
+  deprecated: () => ({
+    points: 1760572800, // October 16th 2025 GMT
```

Comment:
The terminology is `kPoints` rather than `points`

See the example with Dolomite https://github.com/0xPortalLabs/points-adapters?tab=readme-ov-file#deprecated-points 
```suggestion
    kPoints: 1760572800, // October 16th 2025 GMT
```